spot_rl_experiments/spot_rl/envs/gaze_env.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, config, spot: Spot, use_mobile_pick: bool = False):
             max_lin_dist_key=max_lin_dist_key,
             max_ang_dist_key=max_ang_dist_key,
         )
+        self.grasp_dist_threshold = 1.5
```

Comment:
I know I did this, but can we change this to read from the config? perhaps something like GRASP_DIST_MOBILE_GAZE and GRASP_DIST_MOBILE_GAZE_EE so we can use different values for the different policies